_data/starters/rocket11ty.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://github.com/aboutDavid/rocket11ty",
+	"name": "Rocket11ty",
+	"description": "A simple 11ty blog with Halfmoon CSS based on 11ty/eleventy-base-blog",
+	"author": "aboutDavid",
+	"demo": "https://github.com/aboutDavid/rocket11ty#readme"
```

Comment:
You want this to be the live demo of your site!

Reply:
Whoops

Reply:
Ok, fixed!